provider/lxd/environ_network.go

```diff
@@ -80,7 +79,7 @@ func (e *environ) Subnets(ctx context.ProviderCallContext, inst instance.Id, sub
 			// so this call will fail. If that's the case then
 			// use a fallback method for detecting subnets.
 			if isErrMissingAPIExtension(err, "network_state") {
-				return e.subnetDetectionFallback(srv, inst, keepList, azName)
+				return e.subnetDetectionFallback(srv, inst, keepList, availabilityZones)
```

Comment:
What will happen if we upgrade in place or migrate and the az zones don't include the server name, that used to be used?

Reply:
This is not a problem. Before we had a slice of strings with only one address corresponding to the first member's address. Now we'll have a slice of strings with all the addresses of the members (including the one of the first member). I see no issues with it